lib/resources/nginx_module.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+# author: Rony Xavier, rx294@gmail.com
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
+
+module Inspec::Resources
+  class NginxModule < Inspec.resource(1)
+    name 'nginx_module'
+    desc 'Use the nginx_module InSpec audit resource to test nginx modules on Linux platforms.'
+    example "
+      describe nginx_module('http_auth_request') do
+        it { should be_loaded }
+      end
+      "
+
+    def initialize(modulename = nil)
+      @module = modulename
```

Comment:
Like your `nginx` resource, you probably need to take an optional argument for the full path to the `nginx` binary.